magento/adobe-stock-integration
# use MFTF Credentials for storing sensitive variables

... such as api keys and passwords.



### Description (*)
added a new SetCorrectConfig action group which explicitly only leverages the securely stored api and private key for adobe stock. the origin SetConfig action group remains as overrideable with arguments so that failure paths may be explored (i.e. entering invalid api keys). the suite `before` uses the new SetCorrectConfig action. also documented which variables go in `.env` and which go in `.credentials`.

### Relevant Issue

This PR fixes #698.

### Manual testing scenarios (*)

- Follow the updated README directions on how to fill out the two configuration files for MFTF (`.env` and `.credentials`)
- Run one of the MFTF tests that use all four sensitive fields, such as `AdminAdobeStockSignedInCreditsVisibleTest`, i.e.:

      vendor/bin/mftf run:test AdminAdobeStockSignedInCreditsVisibleTest --remove --verbose

- Note that in the command list in the test run output, sensitive fields such as your Adobe IMS email and password are encrypted.
